packages/ratelimits/package.json

```
@@ -0,0 +1,48 @@
+{
+	"name": "@sapphire/ratelimits",
+	"version": "1.0.1",
```

```suggestion
	"version": "0.0.1",
```

version will be set to 1.0.0 by lerna